src/commands/lambda/flare.ts

```diff
@@ -1,14 +1,194 @@
+    if (this.region === undefined) {
+      this.context.stderr.write(renderError('No region specified. [-r,--region]'))
+      errorFound = true
+    }
+
+    // Validate Datadog API key
+    this.apiKey = this.apiKey ?? process.env[CI_API_KEY_ENV_VAR] ?? process.env[API_KEY_ENV_VAR]
+    if (this.apiKey === undefined) {
+      this.context.stderr.write(renderError('No Datadog API key specified. [--api-key]'))
+      errorFound = true
+    }
+
+    // Validate case ID
+    if (this.caseId === undefined) {
+      this.context.stderr.write(renderError('No case ID specified. [-c,--case-id]'))
+      errorFound = true
+    }
+
+    // Validate email
+    if (this.email === undefined) {
+      this.context.stderr.write(renderError('No email specified. [-e,--email]'))
+      errorFound = true
+    }
+
+    if (errorFound) {
+      return 1
+    }
+
+    // Get AWS credentials
+    this.context.stdout.write('\n🔑 Getting AWS credentials...\n')
+    const credentials = await getAWSCredentials()
+    if (credentials === undefined) {
+      this.context.stdout.write(renderSoftWarning("No AWS credentials found, let's set them up!"))
+      await requestAWSCredentials()
+    } else {
+      this.credentials = credentials
+    }
+
+    // Get Lambda function configuration
+    this.context.stdout.write('🔍 Getting Lambda function configuration...\n')
+    const lambdaClientConfig: LambdaClientConfig = {
+      region: this.region,
+      credentials: this.credentials,
+    }
+    const lambdaClient = new LambdaClient(lambdaClientConfig)
+    if (this.functionName === undefined) {
+      throw new Error('Function name is undefined')
+    }
+    const config = await getLambdaFunctionConfig(lambdaClient, this.functionName)
+    const configStrColored = util.inspect(config, false, undefined, true)
+    const configStrUncolored = JSON.stringify(config, undefined, 2)
+
+    // Print config
+    this.context.stdout.write(`\n${configStrColored}\n`)
+    if (this.isDryRun) {
+      this.context.stdout.write('\n🚫 Configuration not sent because the command was ran as a dry run.\n')
+
+      return 0
+    }
+
+    // Write config to file
+    const folderPath = path.join(process.cwd(), '.lambda-flare-output')
+    if (!fs.existsSync(folderPath)) {
+      fs.mkdirSync(folderPath)
+    }
+    const filePath = path.join(folderPath, 'function_config.json')
+    fs.writeFileSync(filePath, configStrUncolored)
+
+    // Zip folder
+    const zipPath = path.join(folderPath, 'lambda-flare-output.zip')
+    const zipFolder = async (command: Command) => {
+      try {
+        const data = await fs.promises.readFile(filePath, 'utf8')
+        const zip = new JSZip()
+        zip.file('function_config.json', data)
+        const content = await zip.generateAsync({type: 'nodebuffer'})
+        await fs.promises.writeFile(zipPath, content)
+      } catch (error) {
+        command.context.stderr.write(renderError('Error reading file or writing zip file. Please try again.'))
```

Comment:
This would output `[Error] Error reading...`. Let's change the wording here so we don't repeat the word Error two times.

Reply:
Good catch!